scripts/tests/dfx_tests.sh

```diff
@@ -97,14 +104,19 @@ dfx ledger fabricate-cycles --t 1000000 --canister $wallet_principal
 
 sleep 10
 
+# run tests
+set +e
```

Comment:
why? this is already set at the top of the file

Reply:
Leftovers for uploading logs to github. Removed